src/styles/utils/_mixin.scss

```diff
@@ -0,0 +1,14 @@
+@mixin squares () {
+  box-sizing: border-box;
```

Comment:
box sizing should be the same for all element, use * selector in normalize file to set it